pandapipes/component_models/pump_component.py

```diff
@@ -150,8 +150,8 @@ def extract_results(cls, net, options, branch_results, nodes_connected, branches
             f, t = get_lookup(net, "branch", "from_to")[cls.table_name()]
             res_table = net["res_" + cls.table_name()]
             if net.fluid.is_gas:
-                p_from = branch_results["p_from"][f:t]
```

Comment:
Did we also produce an error with respect to compressibility at the nodes, since we did not consider the absolute pressure? (cf. line 159)

Reply:
Thanks for pointing this out. The compressibility is calculated on relative pressure, so the ambient pressure needs to be subtracted. I'll fix it

Reply:
For my own understanding of the matter; why is the compressibility calculated for relative pressure ?

Reply:
FYI, on Line 80:
`
p_from = node_pit[from_nodes, PAMB] + node_pit[from_nodes, PINIT]`

Reply:
It should be calculated at abs. pressure, thanks for pointing that out

Reply:
Again for my understanding: There was an error in the calculation before, right?

Reply:
The error in the int ital calculation was that `p_from` and `p_to` were relative pressures but handled like they were absolute pressures.  Sorry for the confusion...